nautobot/ipam/forms.py

```diff
@@ -360,8 +283,16 @@ class Meta:
             "description",
             "tenant_group",
             "tenant",
+            "rir",
+            "date_allocated",
             "tags",
         ]
+        help_texts = {
+            "rir": "Regional Internet Registry responsible for this prefix",
```

Comment:
Should this help_text be defined directly on the Prefix model instead of here in the form?